tests/client/handlers/vaults.test.ts

```diff
@@ -1587,16 +1588,31 @@ describe('vaultsSecretsNewDir and vaultsSecretsList', () => {
       dirName: secretDir,
     });
     expect(addResponse.success).toBeTruthy();
-    // List secrets
-    const listResponse = await rpcClient.methods.vaultsSecretsList({
+
+    await expect(async () => {
+      const files = await rpcClient.methods.vaultsSecretsList({
+        nameOrId: vaultsIdEncoded,
+        path: 'doesntExist',
+      });
+      try {
+        for await (const _ of files); // Consume values
+      } catch (e) {
+        throw e.cause;
+      }
+    }).rejects.toThrow(clientErrors.ErrorClientFSReadFailed);
```

Comment:
it might be a better idea to unwrap this expectation and just do the expect in the try-catch block.

```ts
try {
        for await (const _ of files); // Consume values
        throw Error('failed to throw an error');
      } catch (e) {
        expect(e.cause).toBeInstanceOf(clientErrors.ErrorClientFSReadFailed);
      }
```

or only wrap the `for await (const _ of files);` in the expectation async function.

Reply:
I got the `expect.rejects.toThrow` from another test case, like `tests/vaults/VaultOps.test.ts:226`

```ts
test('updating a secret that does not exist should fail', async () => {
      await expect(
        vaultOps.updateSecret(vault, secretName, secretContentNew),
      ).rejects.toThrow(vaultsErrors.ErrorSecretsSecretUndefined);
    });
```